src/capi.rs

```diff
@@ -218,7 +218,7 @@ impl EncContext {
   fn receive_packet(&mut self) -> Result<Packet, rav1e::EncoderStatus> {
     fn receive_packet<T: rav1e::Pixel>(
       ctx: &mut rav1e::Context<T>,
-    ) -> Result<Packet, rav1e::EncoderStatus> {
+    ) -> Result<Packet, rav1e::EncoderStatus> where FrameInternal: From<Arc<v_frame::frame::Frame<T>>> {
```

Comment:
Why do you need to express that bound?

Reply:
I am not sure, it is pretty gross. The compiler suggested it to me and it doesn't work without. Is there a way to tell it that the From trait is implemented on all the FrameInternal variants?

Reply:
Now I see why, either we do the blanket for every `T: Pixel` or keep this.